metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Publication.java

```diff
@@ -20,28 +20,37 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.annotations.ApiModelProperty;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.validation.constraints.NotNull;
 
 @Entity
-public class Publication extends Auditable<String>{
+public class Publication extends Auditable<Long>{
 
-    @ApiModelProperty(position = 1, value = "PubmedId", required = true)
-    @JsonProperty
-    @NotNull
+    @ApiModelProperty(position = 1, value = "Publication auto generated id", required = true, readOnly = true)
```

Comment:
required=true should be removed

Reply:
Sure, i observe required=true is present across all entities. So are we planning to avoid this in future for all entities?

Reply:
yes we kinda need to remove it for all entitites

Reply:
can you please remove for all entities for consistency